scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -907,7 +907,11 @@
                   ;; freedom in the form of fewer jobs to consider.
                   (if matched-head?
                     max-considerable
-                    (max 1 (long (* scaleback num-considerable))))) ;; With max=1000 and 1 iter/sec, this will take 88 seconds to reach 1
+                    (let [new-considerable (max 1 (long (* scaleback num-considerable)))] ;; With max=1000 and 1 iter/sec, this will take 88 seconds to reach 1
+                      (log/info "Failed to match head, reducing number of considerable jobs" {:prev-considerable num-considerable
+                                                                                              :new-considerable new-considerable})
+                      new-considerable)
+                    ))
```

Comment:
Should we mention the pool in these logs?

Reply:
Done